x/gmp/keeper/msg_server_test.go

```diff
@@ -0,0 +1,38 @@
+package keeper_test
```

Comment:
Should probably add an issue for implementing a RelayMsg integration test in the future